src/ruby/detectors/sensitive-http-action/sensitive-http-action-noncompliant.rb

```diff
@@ -0,0 +1,17 @@
+=begin
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+=end
+
+# {fact rule=sensitive-http-action@v1.0 defects=1}
+class AccountsController < ApplicationController
+    def sensitive_http_get_noncompliant
```

Comment:
Should we merge this with compliant sensitive http action non compliant example?